docs/decisions/0002-forum_v1_migration_to_v2.rst

```diff
@@ -54,6 +54,148 @@ Migration Decisions
+       "pinned" : null,
+       "retired_username" : null,
+       "thread_type" : "discussion",
+       "title" : "a thread about green artichokes",
+       "updated_at" : "2024-09-04T15:06:23Z",
+       "visible" : true,
+       "votes" : {
+         "up" : [],
+         "down" : [],
+         "up_count" : 0,
+         "down_count" : 0,
+         "count" : 0,
+         "point" : 0
+       }
+     }
+   }
+
+  .. code-block:: json
+
+   {
+     "_index" : "comments_20240904150622420",
+     "_type" : "_doc",
+     "_id" : "66d8776f8c6533121e36067d",
+     "_score" : 1.0,
+     "_source" : {
+       "abuse_flaggers" : [],
+       "anonymous" : false,
+       "anonymous_to_peers" : false,
+       "at_position_list" : [],
+       "author_id" : "cordell.mueller_5",
+       "author_username" : "cordell.mueller_5",
+       "body" : "a comment about greed pineapples",
+       "child_count" : null,
+       "comment_thread_id" : "66d8776f8c6533121e36067b",
+       "commentable_id" : "test_commentable",
+       "course_id" : "test/course/id",
+       "created_at" : "2024-09-04T15:06:23Z",
+       "depth" : 0,
+       "endorsed" : false,
+       "endorsement" : null,
+       "historical_abuse_flaggers" : [],
+       "parent_id" : null,
+       "parent_ids" : [],
+       "retired_username" : null,
+       "sk" : "66d8776f8c6533121e36067d",
+       "updated_at" : "2024-09-04T15:06:23Z",
+       "visible" : true,
+       "votes" : {
+         "up" : [],
+         "down" : [],
+         "up_count" : 0,
+         "down_count" : 0,
+         "count" : 0,
+         "point" : 0
+       }
+     }
+   }
+
+* **Example of Optimized Indices in v2**:
+
+  .. code-block:: json
+
+   {
+     "_index" : "comment_threads_20240904095228",
+     "_type" : "_doc",
+     "_id" : "66d8742decb2a86851511014",
+     "_score" : 1.0,
+     "_source" : {
+       "id" : "66d8742decb2a86851511014",
+       "title" : "a thread about green artichokes",
+       "body" : "text",
+       "created_at" : "2024-09-04T09:52:29.049000",
+       "updated_at" : "2024-09-04T09:52:29.049000",
+       "last_activity_at" : "2024-09-04T09:52:29.049000",
+       "comment_count" : 0,
+       "votes_point" : 0,
+       "context" : "course",
+       "course_id" : "course_id",
+       "commentable_id" : "test_commentable",
+       "author_id" : "1",
+       "group_id" : null,
+       "thread_id" : "66d8742decb2a86851511014"
+     }
+   }
+
+  .. code-block:: json
+
+   {
+     "_index" : "comments_20240904095228",
+     "_type" : "_doc",
+     "_id" : "66d8742decb2a86851511015",
+     "_score" : 1.0,
+     "_source" : {
+       "body" : "a comment about greed pineapples",
+       "course_id" : "course_id",
+       "comment_thread_id" : "66d8742decb2a86851511014",
+       "commentable_id" : null,
+       "group_id" : null,
+       "context" : "course",
+       "created_at" : "2024-09-04T09:52:29.067000",
+       "updated_at" : "2024-09-04T09:52:29.068000",
+       "title" : null
+     }
+   }
+
 Conclusion
```

Comment:
we can also add some more points related to elastic search improvements in conclusion.